SYCL/ESIMD/aot_mixed.cpp

```diff
@@ -8,6 +8,7 @@
 // REQUIRES: gpu
 // UNSUPPORTED: cuda || hip
 // UNSUPPORTED: esimd_emulator
+// UNSUPPORTED: gpu-intel-gen9
```

Comment:
```suggestion
// Temporary disable everywhere until "Unsupported required sub group size" is fixed in some configurations.
// UNSUPPORTED: windows
// UNSUPPORTED: linux
```
Should we try that as unconditional disable?

Reply:
Unconditional disable would be the simplest way for now.
"UNSUPPORTED: gpu-intel-gen9" prevents it from running on Gen9, but I guess the error happening on newer Gen, right?